### DISCUSSION TITLE
Why does Firefly show that I overspent when all budgets have money available?

### DISCUSSION THREAD

**Discussion by drvieira** [firefly-iii/firefly-iii#7406] [Firefly III] [answered]
I`'ve been using the budgets feature for a few months and it always worked fine. But this month after a few transactions the top panel changed to red. I checked the budgets chart and they all show some money available and zero overspent.

Is there another reason for the overspent panel that I may be missing?

I've atached a screenshot from my v6.0.7 installation below to better ilustrate what is happening.

![image](https://user-images.githubusercontent.com/98040063/233374524-15755936-a881-4838-b9ea-8fb7d3f04a69.png)

**Comment by JC5** [comment, marked as answer]
What does it say under the /budgets tab? This may happen when you have budgets with several date ranges, as I believe you have.

**Reply by drvieira** [reply]
That was the problem. I had budgets with diferent ranges. This also made me realise that I had too many budgets. Now I have fewer budgets and all of them monthly. Everything is working correctly now.

Many thanks for your reply and also for creating this excelent software!

**Reply by JC5** [reply]
Not a problem! I have it on my list to make sure this is better visible.
